### PR TITLE
impl-lists-tweets-lookup

### LINKED ISSUE #66: List Tweets lookup

### PR DESCRIPTION
**Pull request by mmmommm**
close #66

### CODE REVIEW

File: list.go [hunk 1]
```diff
@@ -50,3 +50,13 @@ type ListLookUpByIDResponse struct {
 	Detail   string              `json:"detail,omitempty"`
 	Type     string              `json:"type,omitempty"`
 }
+
+type ListsTweetsLookUpByIDResponse struct {
+	Tweets   []*Tweet            `json:"data"`
+	Includes *ListIncludes       `json:"includes,omitempty"`
+	Errors   []*APIResponseError `json:"errors,omitempty"`
+	Meta     *ListMeta           `json:"meta"`
+	Title    string              `json:"title,omitempty"`
```
Comment (mmmommm) [inline list.go:59]: Run this command and then invalid request error return this struct.
`curl "https://api.twitter.com/2/lists/1111111111111111111111111111111111111111/tweets" -H "Authorization: Bearer $token"`

File: list_option.go [hunk 2]
```diff
@@ -11,6 +11,7 @@ type ListLookUpOption struct {
 	ListFields      []ListField
 	MaxResults      int
 	PaginationToken string
```
Comment (mmmommm) [inline list_option.go:13]: Query parameters could have `TweetFields`
FYI https://developer.twitter.com/en/docs/twitter-api/lists/list-tweets/api-reference/get-lists-id-tweets

File: list.go [hunk 3]
```diff
@@ -50,3 +50,13 @@ type ListLookUpByIDResponse struct {
 	Detail   string              `json:"detail,omitempty"`
 	Type     string              `json:"type,omitempty"`
 }
+
+type ListsTweetsLookUpByIDResponse struct {
+	Tweets   []*Tweet            `json:"data"`
+	Includes *ListIncludes       `json:"includes,omitempty"`
```
Comment (sivchari) [inline list.go:56]: `Includes` field has `Tweet` field.
Please run
`curl "https://api.twitter.com/2/lists/84839422/tweets?expansions=referenced_tweets.id&tweet.fields=id,source,text,withheld&max_results=10" -H "Authorization: Bearer $token" `

FYI
https://developer.twitter.com/en/docs/twitter-api/lists/list-tweets/api-reference/get-lists-id-tweets#cURL

**Review by sivchari — CHANGES_REQUESTED** [review]

File: list.go (hunk 3 above)
Comment (mmmommm) [reply list.go:57]: Sorry, I missed it.

**Review by sivchari — APPROVED** [review]
LGTM

**Pull request merged**